game/view/actor/minimap.lua

```diff
@@ -0,0 +1,110 @@
+
+local Color      = require 'common.color'
+local SCHEMATICS = require 'domain.definitions.schematics'
+local COLORS     = require 'domain.definitions.colors'
+local FONT       = require 'view.helpers.font'
+
+local min = math.min
+local max = math.max
+
+local _TILE_W = 8
+local _TILE_H = 8
+local _RAD_W = _TILE_W/2
+local _RAD_H = _TILE_H/2
+
+local _TILE_COLORS = {
+  [SCHEMATICS.WALL]  = Color:new {0.3, 0.5, 0.9, 1},
+  [SCHEMATICS.FLOOR] = Color:new {0.1, 0.3, 0.7, 1},
+  [SCHEMATICS.EXIT]  = Color.fromInt {200, 200,  40, 255},
+}
+
+local _width, _height
+local _tile
+local _body
+local _map
+
+local MINIMAP = {}
+
+function MINIMAP.init(width, height)
+  _width, _height = width, height
+  _map = love.graphics.newCanvas(width, height)
+  _tile = {
+    0, 0,
+    1, 0,
+    1, 1,
+    0, 1,
+  }
+  _body = {
+    0.5, 0,
+    1, 0.5,
+    0.5, 1,
+    0, 0.5,
+  }
+  _font = FONT.get("Text", 20)
```

Comment:
Again, some of these behave like constants, so it's better to uppercase them.

Also, the `_tile` and `_body` variables could use a `_polygon` or `_vertices` suffix.